Formula/tinc.rb

```diff
@@ -23,6 +23,41 @@ def install
     system "make", "install"
   end
 
+  plist_options manual: "tincd"
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>ProgramArguments</key>
+          <array>
+            <string>#{sbin}/tincd</string>
+            <string>-D</string>
+            <string>--pidfile</string>
+            <string>#{var}/tincd.pid</string>
+          </array>
+          <key>KeepAlive</key>
+          <dict>
+            <key>NetworkState</key>
+            <true/>
+            <key>Crashed</key>
+            <true/>
+            <key>SuccessfulExit</key>
+            <false/>
+          </dict>
+          <key>StandardErrorPath</key>
+          <string>#{var}/log/tincd.log</string>
+          <key>StandardOutPath</key>
+          <string>#{var}/log/tincd.log</string>
+        </dict>
+      </plist>
+    EOS
+  end
+
   test do
     assert_match version.to_s, shell_output("#{sbin}/tincd --version")
```

Comment:
Could you make a better test for this? I doubt this will catch any problems in the build

Reply:
Checking if the binary builds and can be executed (which I assume the current test does) seems enough for me. This is a vpn client so it would require spinning up a second helper tincd and trying to connect both together. TBH I have no idea how to do this in homebrew. Therefore I'm afraid I will have to leave my patch as it is.

Reply:
Maybe trying to start the tincd using the .plist file would be a good test to have. If you agree I will try to make it so.